### ISSUE TITLE
Denial of Service in is-url-http

### ISSUE THREAD

**Issue by Hbkhan** [Kikobeats/is-url-http#7]
## ✍️ Description

Affected versions of this package are vulnerable to Denial of Service. An attacker providing a very long url can cause a Denial of Service. The vulnerability exists in [#L11](https://github.com/Kikobeats/is-url-http/blob/d7df4f75db122da5a2950640632b0667f5e74651/index.js#L11)

# PoC
```node
const isUrlHttp = require('is-url-http')
isUrlHttp('https://kikobeats.test.3423423423sadasd.1312321321sadsadsad.21312312321asdasdsa.21312321asdasda.21312312312adasd23423.sadasdasd231412321.34234234asdasdasdasdassdasd.34234234asdasdas')
```

**Comment by Kikobeats** [comment]
Please make a PR!

**Comment by Hbkhan** [comment]
@Kikobeats The process is to report a vulnerability to huntr.dev, and once they verify then they post it on their platform, so other developers can look into it and introduce a good fix for the package.

Anyway, I have introduced a fix for you #8. Can you request a CVE for this?
